apps/studio/src/server/modules/permissions/permissions.service.ts

```diff
@@ -111,40 +114,35 @@ export const validateUserPermissionsForResource = async ({
   }
 }
 
-const definePermissionsForManagingUsers = async ({
+export const sitePermissions = async ({
```

Comment:
[nit] could we change this to a verb cos it's doing something

Reply:
simplified in https://github.com/opengovsg/isomer/pull/1136/commits/b1e9924ac15052fa96fff7cbe8de50bcf37e4cc9